include/dt-bindings/gpio/gpio.h

```diff
@@ -76,6 +76,22 @@
 
 /** @} */
 
+/**
+ * @name GPIO pin voltage flags
+ * @{
```

Comment:
This flags are likely not to be supported on per pin basis by other SoC families. Let's be explicit and mention it in the comment here.
```
 * @name GPIO pin voltage flags
 * 
 * Only applicable if SoC allows to configure pin voltage per individual pin.
 * 
 * @{
```

Reply:
Done